src/createExplorer.ts

```diff
@@ -255,10 +255,10 @@ class Explorer {
     content: string | null,
   ): Promise<CosmiconfigResult> {
     return Promise.resolve()
-      .then(() => {
+      .then(():Promise<LoadedFileContent> | LoadedFileContent  => {
```

Comment:
Should there be a space after this colon?

Reply:
I ran prettier after all commits to minimize changes per commit. This comment is marked outdated so I’m going to assume prettier did it’s job. (I’m on mobile right now, I’ll check for sure when I’m on a computer.

Reply:
Just checked, this was fixed in https://github.com/davidtheclark/cosmiconfig/pull/202/commits/19fc5ef3fd8d1bde07aee619ea00bbe2ea804429.